ensembl/htdocs/info/genome/compara/species_trees.html

```diff
@@ -17,19 +17,72 @@ <h1 id="speciestrees">Species trees</h1>
+
+    <li>This consensus species tree is then used as a starting topology for a refinement process. A supermatrix is created by concatenating the protein alignments of all genes and it is trimmed by using TrimAl <a href="#fn7">[7]</a> (version v1.4.rev15). Then IQ-TREE is used to performs a new, full tree search on this supermatrix to find the optimal species tree topology, using the Astral tree as the starting topology. This step refines both the topology and branch lengths of the final tree.</li>
+
+    <li>Finally, branch lengths for this definitive topology are also computed from the fourfold degenerate sites of the concatenated codon alignment (used in Cactus <a href="#fn8">[8]</a> multiple genome alignments).</li>
+    </ol>
+
     </li>
 </ol>
 
+<h2 id="notes">Notes and References</h2>
+
+<ol>
+
+<li id="fn1">
+Manni M, Berkeley MR, Seppey M, Simão FA, Zdobnov EM. BUSCO Update: Novel and Streamlined Workflows along with Broader and Deeper Phylogenetic Coverage for Scoring of Eukaryotic, Prokaryotic, and Viral Genomes.  <a href="https://europepmc.org/article/MED/34320186">Molecular Biology and Evolution, Volume 38, Issue 10, October 2021, Pages 4647–4654</a>.
+</li>
+
+<li id="fn2">
+Li H. Protein-to-genome alignment with miniprot. <a href="https://europepmc.org/article/MED/36648328">Bioinformatics. 2023 Jan;39(1):btad014</a>.
+</li>
+
+<li id="fn3">
+Löytynoja A, Vilella AJ, Goldman N. Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm. <a href="https://europepmc.org/article/MED/22531217">Bioinformatics. 2012 Jul;28(13):1684-1691</a>.
+</li>
+
+<li id="fn4">
+Löytynoja A, Vilella AJ, Goldman N. Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm. <a href="https://europepmc.org/article/MED/22531217">Bioinformatics. 2012 Jul;28(13):1684-1691</a>.
+</li>
+
+<li id="fn5">
+Minh BQ, Schmidt HA, Chernomor O, Schrempf D, Woodhams MD, von Haeseler A, Lanfear R. IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era. <a href="https://europepmc.org/article/MED/32011700">Mol Biol Evol. 2020 May;37(5):1530-1534</a>.
+</li>
+
+<li id="fn6">
+Mirarab S, Warnow T. ASTRAL-II: coalescent-based species tree estimation with many hundreds of taxa and thousands of genes. <a href="https://europepmc.org/article/MED/26072508">Bioinformatics. 2015 Jun;31(12):i44-52</a>.
+</li>
+
+<li id="fn7">
+Capella-Gutiérrez S, Silla-Martínez JM, Gabaldón T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. <a href="https://europepmc.org/article/MED/19505945">Bioinformatics. 2009 Aug;25(15):1972-1973</a>.
+</li>
+
+<li id="fn8">
```

Comment:
In conjunction with the deduplication of Löytynoja et al. (2012):
```suggestion
<li id="fn7">
```